### PR TITLE
paragon-camptune :latest (new cask)

### PR DESCRIPTION
**Pull request by yurikoles**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/paragon-camptune.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "paragon-camptune" do
+  version :latest
+  sha256 :no_check
+
+  url "https://dl.paragon-software.com/demo/camptune_demo_u.dmg"
```
Comment (SMillerDev) [inline Casks/paragon-camptune.rb:5]: From the website this seems like https://docs.brew.sh/Acceptable-Casks#trial-and-freemium-versions

Comment (yurikoles) [reply Casks/paragon-camptune.rb:5]: It can be activated via a Paragon account.

**Comment by yurikoles** [comment]
`createzap` reports that `no settings found`, I'll try to search for then manually

**Comment by miccal** [comment]
@yurikoles also please try to add a [`livecheck` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-livecheck).

**Comment by yurikoles** [comment]
> @yurikoles also please try to add a [`livecheck` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-livecheck).

Check for what? It's unversioned.

**Comment by bevanjkay** [comment]
@yurikoles Can you confirm if there were any files to zap here?

**Comment by yurikoles** [comment]
I didn't find any files to zap.

**Pull request merged**